CONTRIBUTING.md

```diff
@@ -0,0 +1,73 @@
+# How to Contribute
+
+Winston is one of Gorilla Logic projects which is under active development and is also being used to provide information to all collaborators about general inquiries like available PTO, nearby wellness events and others. Hopefully this document makes the process for contributing clear and answers some questions that you may have.
+
+## Open Development
+
+All work on WINSTON-LAMBDA happens directly on GitHub. Both core team members and other contributors send pull requests which go through a code review process.
+
+## Branch Organization
+
+We will do our best to keep the master branch in good shape, with tests passing at all times.
+
+If we happen to find a bug or have ideas for new functionality, we will create a Github Issue, this issue will have an auto-generated number, which we will use to create our branch name with the following structure:
+
+- Issue: #8
+- Branch to work that issue: `GH-8`
+- All commits should have `GH-8` at the beginning, so issues and commits are integrated
+
+If you send a pull request, please do it against the master branch.
+
+## Semantic Versioning
+
+WINSTON-LAMBDA follows semantic versioning. We release patch versions for bugfixes, minor versions for new features, and major versions for any breaking changes. Be sure to update the `package.json` version on every deploy.
+
+## Bugs
+
+### Where to Find Known Issues
+
+We are using GitHub Issues for our public bugs. We keep a close eye on this and try to make it clear when we have an internal fix in progress. Before filing a new task, try to make sure your problem doesn’t already exist.
+
+### Reporting New Issues
+
+The best way to get your bug fixed is to provide a reduced test case. Feel free to create a Github Issue with your bug description and steps to reproduce.
+
+## How to Get in Touch
+
+Contact any of the core contributors, you can find them on `CONTRIBUTORS.md`
+
+## Proposing a Change
+
+If you intend to change the bot, or make any non-trivial changes to the implementation, we recommend filing an issue. This lets us reach an agreement on your proposal before you put significant effort into it.
```

Comment:
"we recommend filing" -> "we recommend **filling**"

Reply:
Filing as in 
> Filing a Police Report 
I think I'll leave this one as-is.